scss/components/_case-peoples.scss

```diff
@@ -17,14 +17,19 @@
 
   .select2-choice {
     height: 30px;
-    padding: 2px 0 2px 10px;
+    padding: 5px 0 5px 10px;
```

Comment:
select2 is used everywhere in the extension, why did the padding need fixing only on this particular page?

Reply:
Valid point. Checking in deep, I found out that we probably don't need to style the select 2 specifically for the people's tab. So we have removed the occurrence of `.select2-` classes from the _case-peoples.scss.